py/desisim/scripts/quickgen.py

```diff
@@ -176,6 +179,34 @@ def main(args=None):
     qsim.atmosphere.airmass = simspec.header['AIRMASS']
     qsim.observation.exposure_time = simspec.header['EXPTIME'] * u.s
 
+    # Set moon parameters
+    if simspec.flavor == 'bgs' or simspec.flavor == 'mws' or simspec.flavor == 'bright':
+        if args.moonphase is None:
+            qsim.atmosphere.moon.moon_phase = 0.7
+        else:
+            qsim.atmosphere.moon.moon_phase = args.moonphase
+        if args.moonangle is None:
+            qsim.atmosphere.moon.separation_angle = 50 * u.deg
+        else:
+            qsim.atmosphere.moon.separation_angle = args.moonangle * u.deg
+        if args.moonzenith is None:
+            qsim.atmosphere.moon.moon_zenith = 30 * u.deg
+        else:
+            qsim.atmosphere.moon.moon_zenith = args.moonzenith * u.deg
+    elif simspec.flavor == 'gray' or simspec.flavor == 'grey':
+        if args.moonphase is None:
+            qsim.atmosphere.moon.moon_phase = 0.1
+        else:
+            qsim.atmosphere.moon.moon_phase = args.moonphase
+        if args.moonangle is None:
+            qsim.atmosphere.moon.separation_angle = 60 * u.deg
+        else:
+            qsim.atmosphere.moon.separation_angle = args.moonangle * u.deg
+        if args.moonzenith is None:
+            qsim.atmosphere.moon.moon_zenith = 80 * u.deg
+        else:
+            qsim.atmosphere.moon.moon_zenith = args.moonzenith * u.deg
+
```

Comment:
Let's add an explicit `else` clause here to set the parameters for the moon below the horizon.  That happens to be the current specsim defaults, but it would be better for quickgen and quickbrick to control their own default behavior so that a future change in specsim doesn't accidentally silently change the quickgen/quickbrick action for dark time targets.